### PR TITLE
Made descriptions closer to what users look for.

### PR DESCRIPTION
**Pull request by angelgolfer-ms**

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[30ca2ef](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4192/commits/30ca2ef8f2d08a47bcb1477500780665aa4899a6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/index.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-fx-landing/concepts/index.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C18%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903181954320437-4192%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2BusNs671s0XIxgmrW9RXp01XerT5NmjTwCKsdeSyceY%3D&st=2019-03-18T19%3A53%3A08Z&se=2019-04-18T19%3A58%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: concepts/index.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ sections:
       columns: 3
       items:
       - href: /graph/overview
-        html: <p>Learn more about Microsoft Graph and its features and capabilities.</p>
+        html: <p>Learn what apps can do in Microsoft Graph and see popular APIs.</p>
```
Comment (Lauragra) [inline concepts/index.yml]: How about something like: "Find out what you can do with Microsoft Graph and see common API requests."

Comment (angelgolfer-ms) [reply concepts/index.yml]: To new customers, the "request" part may be ambiguous if they're not thinking about REST APIs and REST requests. Would they think of "requests" as in what they'd like and can request us to implement?

Comment (Lauragra) [reply concepts/index.yml]: Maybe we need to change the heading in the Overview topic. So, we could say: "Find out what you can do with Microsoft Graph and see common API scenarios."

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[8c07df1](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4192/commits/8c07df1c9e45f04c56711ff150c14a445d84993a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/index.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-fx-landing/concepts/index.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C19%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903190036147627-4192%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=K6PC1U2wFw4Ct%2BfZ6DsZg6QjG%2FEAR9gWhDX3xRNzaSI%3D&st=2019-03-19T00%3A34%3A37Z&se=2019-04-19T00%3A39%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ffeffcb](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4192/commits/ffeffcb81b52766a40d47238bc0b00e86a1210bb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/index.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-fx-landing/concepts/index.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C19%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903190048431734-4192%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=rVoNlRzYryVhrquqPnZe1dKt%2F1ChHQitLW4ke2igPrk%3D&st=2019-03-19T00%3A46%3A59Z&se=2019-04-19T00%3A51%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[33c45e8](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4192/commits/33c45e84e03686a92589a77acbbdd467130c0b33)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/index.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-fx-landing/concepts/index.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C19%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903191814590876-4192%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FRStSpnVG7KElXpFuSIgE7SI3tUE8KdlZQmcSy4EtHc%3D&st=2019-03-19T18%3A13%3A35Z&se=2019-04-19T18%3A18%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: concepts/index.yml (hunk 1 above)
Comment (angelgolfer-ms) [reply concepts/index.yml]: I am going to change the "overview" blurb to "Find out what you can do with  Microsoft Graph and see some popular API use cases". 
The "what you can do with Microsoft Graph" phrase maps to the [section](https://docs.microsoft.com/en-us/graph/overview?view=graph-rest-beta#what-can-you-do-with-microsoft-graph) that describes _scenarios_ in plain English, and the "popular API use cases" phrase maps to the [table](https://docs.microsoft.com/en-us/graph/overview?view=graph-rest-beta#popular-requests) that has some basic API calls (for use cases for user, group, mail, calendar, people, insights, OneNote). 
That table is a subset of basic use cases and is dated. Hopefully  "some popular API use cases" can get us around giving the impression "that's all we can support"?
I'll track updating that topic with what will be in v1 by Build.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[913f6ec](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4192/commits/913f6ecb935ec81e4f9a369931cbf2e98bf050c1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/index.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-fx-landing/concepts/index.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C19%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903191845498740-4192%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=DorECPhkTzF19UP0ViblCS%2F5lr5hzTUE6qONzNQq0bo%3D&st=2019-03-19T18%3A44%3A15Z&se=2019-04-19T18%3A49%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.
